gradle/docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  strongbox-web-integration-tests-nuget:
+    image: strongboxci/alpine:jdk8-gradle-4.5
+    volumes:
+      - $HOME/.m2/repository:/home/jenkins/.m2/repository
+      - ./:/workspace
```

This will result in a fatal error because it can't find the parent pom:
```
[FATAL] Non-resolvable parent POM for org.carlspring.strongbox:strongbox-web-integration-tests-gradle:[unknown-version]: Could not find artifact org.carlspring.strongbox:strongbox-web-integration-tests-parent:pom:1.0-SNAPSHOT in carlspring (http://dev.carlspring.org/nexus/content/groups/carlspring) and 'parent.relativePath' points at wrong local POM @ line 8, column 13
```


```suggestion
      - ../:/workspace
```